core/src/main/java/com/google/googlejavaformat/java/LineSeparator.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
Minor nit: this should read `Copyright 2016 Google Inc.` for consistency with the other source files.